ui/src/app/pages/admin/disks-table.spec.tsx

```diff
@@ -0,0 +1,200 @@
+import '@testing-library/jest-dom';
+
+import * as fp from 'lodash/fp';
+
+import {
+  AppType,
+  Disk,
+  DiskStatus,
+  DiskType,
+  ListDisksResponse,
+} from 'generated/fetch';
+
+import { screen } from '@testing-library/dom';
+import {
+  fireEvent,
+  render,
+  waitFor,
+  waitForElementToBeRemoved,
+  within,
+} from '@testing-library/react';
+import * as swaggerClients from 'app/services/swagger-fetch-clients';
+import moment from 'moment';
+
+import { DisksTable } from './disks-table';
+
+const convertDate = (originalDate: String) => {
+  const date = originalDate.substring(0, 10);
+  const hour =
+    parseInt(originalDate.substring(11, 13), 10) + moment().utcOffset() / 60;
+  const minutes = originalDate.substring(14, 16);
+  return `${date} ${hour}:${minutes}`;
+};
```

Comment:
I was originally trying to avoid using moment to make it independent of the libraries that we were using in the component. It looks like one usage snuck in when I got the offset.With all that said, yeah, this would be cleaner. Thanks!

Reply:
I agree that it's undesirable to use library X to test library X, so I'm fine with either approach.